src/platform/backends/shared/base_virtual_machine_factory.h

```diff
@@ -109,4 +111,8 @@ inline void multipass::BaseVirtualMachineFactory::require_snapshots_support() co
     throw NotImplementedOnThisBackendException{"snapshots"};
 }
 
+inline void multipass::BaseVirtualMachineFactory::require_suspend_support() const
```

Comment:
Also, it should be pretty straight forward to write a unit test for this and show that it doesn't throw an exception.